### ISSUE TITLE
--url flag errors running migration

### ISSUE THREAD

**Issue by redeemefy** [sequelize/cli#1109]

**Comment by WikiRik** [comment]
The CLI does not like the format of your url. It is being parsed in the config-helper and `url.parse` can't find a pathname so it also is unable to replace it.
In the future we should improve the error being returned for this, but to fix this you should provide the name of the database at the end of your url.
https://github.com/sequelize/cli/blob/v6.4.1/src/helpers/config-helper.js#L171-L193

**Comment by redeemefy** [comment]
Thanks for your response. Indeed, the `--url` flag needs the database name at the end of the string. I ran the command and it does run the migration. However, now I'm running the command to rollback the migration following docs [here](https://sequelize.org/docs/v6/other-topics/migrations/#undoing-migrations)

```shell
$ npx sequelize-cli db:migrate:undo --url 'postgres://postgres:somepassword@some-playground-instance-cluster.some-hash.us-east-1.rds.amazonaws.com/dbname'
```

and I'm getting the following:

```shell
ERROR: Unable to find migration: timestamp-migration-file-name.js
```

The file does exists in the database and exist in my folder structure since what able to run the migration a minute ago.

**Comment by WikiRik** [comment]
Does `npm run migrate:up -- --url <url>` work for the first script?

**Comment by redeemefy** [comment]
> 

Yes it does. Not including the flag in `package.json` and modifying the command for `npm run migrate:up -- --url <url>` does work

**Comment by redeemefy** [comment]
I'm creating a PR for updating docs. Against what branch should the PR go.
